share/man/man9/intro.9

```diff
@@ -346,7 +347,7 @@ To voluntarily yield the processor, use
 The various functions which will deliberately put a thread to sleep are
 described by
 .Xr sleep 9 .
-Sleeping threads are removed from the scheduler and placed on a
+Sleeping threads are removed from the scheduler and placed in a
```

Comment:
No, "on" is accurate. Threads are placed _on_ a sleepqueue just like they are placed _on_ a scheduler runqueue.

Reply:
Reverted by [`0c4471a` (#813)](https://github.com/freebsd/freebsd-src/pull/813/commits/0c4471aaeecbc50260cd4310042441e838519ba6).